config/routes.rb

```diff
@@ -52,6 +53,8 @@
     end
   end
 
+  mount GrapeAPI::RootAPI => "/"
```

Comment:
I see that with this we have the following:

- Files have to go in `app/api/grape_api`, while I'd put it in `lib/api`.
- It's not versioned.

Take a look at Gitlab's `lib/api/api.rb` file, since it's closer to what I'd want to accomplish.